packages/components/src/ButtonGroup/ButtonGroup.module.css

```diff
@@ -0,0 +1,48 @@
+.buttonGroup {
```

Comment:
Well've also need to update ButtonGroup to use this and remove the old SCSS module for buttonGroup